### ISSUE TITLE
Hide `left` and `right` strings from translation in 2.3

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#292]
From @RVieyra over slack: A user reported that there are unused strings that look buggy in the translation utility for this sim. See "left" and "right" and how `left` is incorrectly shown twice.
<img width="261" alt="Screen Shot 2022-05-24 at 11 10 35 AM (1)" src="https://user-images.githubusercontent.com/6396244/170101231-ed51180c-c40a-4b2c-9814-589ccaa0f701.png">
 
These strings are currently unused and do not appear in master strings file. They were added for prototype accessibility features in 2017 and leaked into the translatable strings in that time.

Back then we marked a11y strings as hidden from the translation utility with a `visible: false` flag in the json strings file. This flag is used for several strings in the 2.3 branch but not these two. A maintenance release hiding these strings with this flag should be simple.
